PEUQSE/plotting_functions.py

```diff
@@ -140,20 +142,20 @@ def sampledParameterHistogramMaker(parameterSamples, parameterName, parameterNam
                 sampledParameterAxesDictionary[parameterName].set_ylabel('Probability Density', **axis_font)
         sampledParameterAxesDictionary[parameterName].set_xlabel(parameterNamesAndMathTypeExpressionsDict[parameterName], **axis_font)
 
-        if histogram_plot_settings['show_initial_value'] == True:
-            if parameterInitialValue!=None:
-                sampledParameterAxesDictionary[parameterName].axvline(x=parameterInitialValue, color='#00A5DF', linestyle='--')
-        if histogram_plot_settings['show_MAP_value'] == True:
-            if parameterMAPValue!=None:
-                sampledParameterAxesDictionary[parameterName].axvline(x=parameterMAPValue, color='r', linestyle='--')
-        if histogram_plot_settings['show_muAP_value'] == True:
-            if parameterMuAPValue!=None:
-                sampledParameterAxesDictionary[parameterName].axvline(x=parameterMuAPValue, color='k', linestyle='--')
+        # Add vertical lines at initial value, MAP, and, muAP if determined by UserInput
+        # Make sure that the value passed has a value
+        if histogram_plot_settings['show_initial_value'] and type(parameterInitialValue) != type(None):
+            sampledParameterAxesDictionary[parameterName].axvline(x=parameterInitialValue, color='#00A5DF', linestyle='--', linewidth=histogram_plot_settings['vertical_linewidth'])
+        if histogram_plot_settings['show_MAP_value'] and type(parameterMAPValue) != type(None):
```

Comment:
Since you have two comparisons in the same line, in python I always put things in parenthesis now so I don't have to remember the rules:

if (histogram_plot_settings['show_MAP_value']==True) and (type(parameterMAPValue) != type(None)):

Reply:
Note: I think there are a couple of other places in your code with two comparisons in the same line. Please put parentheses around them, too.